src/components/tests/negotiations-panel.test.tsx

```diff
@@ -110,6 +110,8 @@ it('renders negotiations correctly', () => {
     expect(classes?.label?.includes("forfeited"));
   }
 
+  treeItem.at(1).simulate('click');
```

Comment:
You can remove this, since we are not expecting anything.

Reply:
removed 454e67a